spec/features/sending_invitations_spec.rb

```diff
@@ -22,7 +18,7 @@ def fill_out_all_fields
 
   scenario 'not filling out all fields' do
     fill_out_all_fields
-    fill_out_placeholder 'e-mail respondent', ''
+    fill_in 'respondentEmail', with: ''
```

Comment:
Ik heb wel for toegevoegd aan het label, maarnom deze styling mogelijk te maken staat het label naast de input inplaats van eromheen. Dus dan kan ik hier niet het label for gebruiken.